src/jobsautoreport/main.py

```diff
@@ -43,7 +46,11 @@ def main() -> None:
                 "type": "section",
                 "text": {
                     "type": "mrkdwn",
-                    "text": f"• Jobs success rate: _{success_rate}_\n• Number of jobs triggered: _{number_of_jobs_triggered}_\n• Jobs average duration: _{average_job_duration}_",
+                    "text": f"""
+• Jobs success rate: _{success_rate}_
+• Number of jobs triggered: _{number_of_jobs_triggered}_
+• Jobs average duration: _{int(average_jobs_duration.total_seconds()) // 60}_ minutes, and _{int(average_jobs_duration.total_seconds())  % 60}_ seconds
+""",
```

Comment:
```suggestion
                    "text": (f"• Jobs success rate: _{success_rate}_\n"
                                  f"• Number of jobs triggered: _{number_of_jobs_triggered}_\n"
                                  f"• Jobs average duration: _{int(average_jobs_duration.total_seconds()) // 60}_ minutes, and _{int(average_jobs_duration.total_seconds())  % 60}_ seconds")
```
I missed that, I would be cleaner to have multiline in this way as it keeps the indented property,

Reply:
the suggestion is bad because I cannot indent properly in the comment field :-(